apps/native/app/package.json

```diff
@@ -92,26 +97,19 @@
     "@babel/core": "^7.20.0",
     "@babel/preset-env": "^7.20.0",
     "@babel/runtime": "^7.20.0",
-    "@tsconfig/react-native": "^2.0.2",
+    "@tsconfig/react-native": "3.0.5",
     "@types/intl": "^1",
     "@types/jest": "^29.2.1",
     "@types/lodash": "4.17.1",
-    "@types/react": "^18.0.24",
-    "@types/react-dom": "^18",
+    "@types/react": "^18.2.6",
     "@types/react-test-renderer": "^18.0.0",
-    "@types/styled-components": "^5",
-    "@types/styled-components-react-native": "5.2.1",
-    "babel-jest": "^29.2.1",
+    "babel-jest": "^29.6.3",
     "babel-loader": "^8.3.0",
     "babel-plugin-formatjs": "10.3.9",
+    "babel-plugin-module-resolver": "5.0.2",
     "jest": "29.7.0",
-    "metro-react-native-babel-preset": "0.72.3",
-    "react-dom": "18.2.0",
     "react-test-renderer": "18.2.0",
-    "typescript": "4.8.4"
-  },
-  "jest": {
-    "preset": "react-native"
+    "typescript": "5.1.6"
```

Comment:
FYI Gulli is upgrading the monorepo and Nx and that brings typescript 5.4.5. Don't know if you want to check if the app can match that?

Reply:
Thanks for the comment. But the recommended version for this RN version is the current one